benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -860,7 +860,21 @@ void GraphicsBenchmarkApp::Render()
     }
     submitInfo.pFence = frame.renderCompleteFence;
 
+    Timer timerSubmit;
+    timerSubmit.Start();
```

Comment:
Start() does seem to return a result, perhaps worth asserting it is a success?

Reply:
Sound good to me. Updated!